Jesse-Kramer/tumi-mundo
# 70 add languages svgs to language page

Resolves fdnd-agency/tumi-mundo#70

## What does this change?

- Added the svg's for the language flags
- Adjusted the gap/margin between li items
- Made final styling adjustments

## How Has This Been Tested?


![image](https://github.com/user-attachments/assets/a755f93f-91de-4fdc-9be4-080aa976787f)

![image](https://github.com/user-attachments/assets/935f0d7b-c577-4957-aa93-ab5521cb2356)


## Images

![image](https://github.com/user-attachments/assets/173c053d-630f-411a-9582-6eaba35504b8)



## How to review

1. Pull the branch locally.  
2. Run the app and navigate to the page with the languages with "/languages"
3. Verify:
   - Languages render correctly
   - Svg's render correctly
   - Search form is functional
